syft/pkg/cataloger/java/test-fixtures/java-builds/Makefile

```diff
@@ -72,7 +72,7 @@ $(PKGSDIR)/gcc-amd64-darwin-exec-debug:
 	./build-example-macho-binary.sh $(PKGSDIR)
 
 # we need a way to determine if CI should bust the test cache based on the source material
-$(PKGSDIR).fingerprint: clean-examples
-	mkdir -p $(PKGSDIR)
-	find example-* build-* Makefile -type f -exec sha256sum {} \; | sort | tee /dev/stderr | tee $(PKGSDIR).fingerprint
-	sha256sum $(PKGSDIR).fingerprint
+.PHONY: cache.fingerprint
+cache.fingerprint:
```

Comment:
I don't think this fundamentally changed anything -- that is, I think this PR now passes since the fingerprint has changed, but ultimately the cache won't be automatically busted when there is a source change.